Description: Document Debian-specific XPM files location
 Add a section about usage of translated XPM files in Debian.
Author: Sandro Tosi <matrixhasu@gmail.com>

Index: wmclock-1.0.14/wmclock.man.in
===================================================================
--- wmclock-1.0.14.orig/wmclock.man.in	2011-06-02 12:11:40.157887000 +0200
+++ wmclock-1.0.14/wmclock.man.in	2011-06-02 12:11:45.337865330 +0200
@@ -160,6 +160,21 @@
 the bottom.  The weekday abbreviations must also be arranged
 vertically, beginning with Monday at the top and continuing to Sunday
 at the bottom.
+.PP
+On Debian systems you can find XPM files for a variety of languages in: 
+.PP
+.RS
+.BR /usr/share/wmclock/
+.RE
+.PP
+For example, to have a French display, you could use the following 
+command line:
+.PP
+.RS
+.TQ
+wmclock \-monthxpm /usr/share/wmclock/lang.french/month.xpm \-weekdayxpm /usr/share/wmclock/lang.french/weekday.xpm
+.RE
+.PP
 .SS Obsolete Options
 .PP
 In order to maintain command-line compatibility (mostly) with
